### PR TITLE
Fix some timeouts/crashes on json testing 

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #7855 (issue introducing use of json for docs)

### CODE REVIEW

**Comment by GiudGiud** [comment]
just a bunch of prereq. The test does a syntax dump, and if the names are the same that s not great. 

alternate fixes:
- stop parallel scheduling on the entire outputs folder
- pass a name to rename the dump, though not sure what yet

File: test/tests/outputs/format/tests [hunk 1]
```diff
@@ -77,47 +77,48 @@
...
     type = 'RunApp'
     input = 'pps_screen_out_warn.i'
-    expect_out = "A Postprocessor utilizes the 'outputs' parameter; however, postprocessor output is disabled for the 'console' output object."
+    expect_out = "A Postprocessor utilizes the 'outputs' parameter; however, postprocessor output is "
+                 "disabled for the 'console' output object."
     allow_warnings = true
     max_parallel = 1 # warning can mix on multiple processes
     issues = '#1426'
     design = 'syntax/Outputs/index.md'
     requirement = 'If a postprocessor utilizes the "outputs" parameter, but postprocessor output is disabled for the console object, the system shall warn the user.'
```
Comment (cticenhour) [inline test/tests/outputs/format/tests:118]: I know this isn't part of this PR, but can we fix this while here? 
```suggestion
    requirement = 'The system shall warn the user if a postprocessor utilizes the "outputs" parameter but postprocessor output is disabled for the console object.'
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
I'm kinda OK with using the prereqs param to address this, though perhaps making sure they don't generate output with the same name is better if these tests don't *really* rely on each other. I did notice a couple things I wanted to fix up re: SQA while here though.

File: test/tests/outputs/format/tests [hunk 2]
```diff
@@ -47,28 +47,28 @@
...
 
-  [./tecplot_bin_test]
+  [tecplot_bin_test]
     type = 'CheckFiles'
     input = 'output_test_tecplot_binary.i'
     check_files = 'output_test_tecplot_binary_out_0000.plt'
     tecplot = true
     issues = '#3403'
     design = 'Tecplot.md'
     requirement = 'If the Tecplot API is available, the system shall support Tecplot output given in Tecplot binary format.'
```
Comment (cticenhour) [inline test/tests/outputs/format/tests:68]:
```suggestion
    requirement = 'The system shall support Tecplot output given in Tecplot binary format if the Tecplot API is available.'
```

File: test/tests/outputs/format/tests [hunk 3]
```diff
@@ -77,47 +77,48 @@
     design = 'Tecplot.md'
     issues = '#3403'
     requirement = 'If the Tecplot API is not available, the system shall support Tecplot output given in ASCII format.'
```
Comment (cticenhour) [inline test/tests/outputs/format/tests:79]:
```suggestion
    requirement = 'The system shall support Tecplot output given in ASCII format if the Tecplot API is not available.'
```

File: test/tests/outputs/format/tests [hunk 4]
```diff
@@ -126,97 +127,101 @@
...
-  [../]
+  []
 
-  [./sln_out_test]
+  [sln_out_test]
     type = 'CheckFiles'
     input = 'output_test_sln.i'
     check_files = 'output_test_sln_out.slh'
     design = 'SolutionHistory.md'
     requirement = 'Solution history output shall work'
```
Comment (cticenhour) [inline test/tests/outputs/format/tests:137]:
```suggestion
    requirement = 'The system shall support solution history output.'
```

**Review by cticenhour — COMMENTED** [review]
A few more. Thanks for allowing these to ride along with this PR.

**Review by cticenhour — APPROVED** [review]

**Pull request merged**